logic/atrium-logic-common/src/main/kotlin/ch/tutteli/atrium/logic/creating/collectors/AssertionsOptionExplantoryExtensions.kt

```diff
@@ -3,18 +3,35 @@ package ch.tutteli.atrium.logic.creating.collectors
 import ch.tutteli.atrium.assertions.AssertionGroup
 import ch.tutteli.atrium.assertions.ExplanatoryAssertionGroupType
 import ch.tutteli.atrium.assertions.builders.AssertionsOption
+import ch.tutteli.atrium.assertions.builders.ExplanatoryAssertionGroupFinalStep
```

Comment:
this deprecation needs to be suppressed as well (add deprecation to `@file`) this should fix the build